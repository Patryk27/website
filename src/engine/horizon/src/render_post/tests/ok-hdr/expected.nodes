Elem(
    Spanned {
        inner: Elem {
            name: Spanned {
                inner: "hdr",
                span: Span {
                    start: 1,
                    end: 4,
                },
            },
            attrs: [
                Attr {
                    name: Spanned {
                        inner: "id",
                        span: Span {
                            start: 5,
                            end: 7,
                        },
                    },
                    value: Some(
                        Spanned {
                            inner: "hello-world",
                            span: Span {
                                start: 8,
                                end: 21,
                            },
                        },
                    ),
                },
            ],
            children: [
                Text(
                    Spanned {
                        inner: "\n  Hello, ",
                        span: Span {
                            start: 22,
                            end: 32,
                        },
                    },
                ),
                Elem(
                    Spanned {
                        inner: Elem {
                            name: Spanned {
                                inner: "i",
                                span: Span {
                                    start: 33,
                                    end: 34,
                                },
                            },
                            attrs: [],
                            children: [
                                Text(
                                    Spanned {
                                        inner: "World",
                                        span: Span {
                                            start: 35,
                                            end: 40,
                                        },
                                    },
                                ),
                            ],
                        },
                        span: Span {
                            start: 32,
                            end: 44,
                        },
                    },
                ),
                Text(
                    Spanned {
                        inner: "!\n",
                        span: Span {
                            start: 44,
                            end: 46,
                        },
                    },
                ),
            ],
        },
        span: Span {
            start: 0,
            end: 52,
        },
    },
)
Text(
    Spanned {
        inner: "\n",
        span: Span {
            start: 52,
            end: 53,
        },
    },
)