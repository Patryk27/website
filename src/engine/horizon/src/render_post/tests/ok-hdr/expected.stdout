<h2 id="hello-world"><a href="#hello-world">
  Hello, <i>World</i>!
</a></h2>
